specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2022-10-01/updates.json

```diff
@@ -0,0 +1,387 @@
+            "schema": {
+              "$ref": "#/definitions/Update"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a specific update": {
+            "$ref": "./examples/GetUpdates.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "UpdateList": {
+      "description": "List of Updates",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of Updates",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Update"
+          }
+        }
+      }
+    },
+    "UpdatePrerequisite": {
+      "description": "If update State is HasPrerequisite, this property contains an array of objects describing prerequisite updates before installing this update. Otherwise, it is empty.",
+      "type": "object",
+      "properties": {
+        "updateType": {
+          "description": "Updatable component type.",
+          "type": "string",
+          "x-ms-client-flatten": true
+        },
+        "version": {
+          "description": "Version of the prerequisite.",
+          "type": "string",
+          "x-ms-client-flatten": true
+        },
+        "packageName": {
+          "description": "Friendly name of the prerequisite.",
+          "type": "string",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "Update": {
+      "description": "Update details",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "location": {
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "description": "The geo-location where the resource lives"
+        },
+        "properties": {
+          "description": "Update properties",
+          "type": "object",
+          "$ref": "#/definitions/UpdateProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "UpdateProperties": {
+      "description": "Details of a singular Update in HCI Cluster",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning state of the Updates proxy resource.",
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Accepted",
+            "Provisioning"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "installedDate": {
```

Comment:
>installedDate [](http://example.com/codeflow?start=9&length=13)

Is there a difference between "install" and "apply"?

Reply:
No

Reply:
Got it. Then this property should go in the systemData.